### PR TITLE
Updated prompt evaluation notebook

### PR DESCRIPTION
**Pull request by Hari-Krishnan3**
aiopenscale/tracker#33955

### CODE REVIEW

File: IBM Cloud/WML/notebooks/watsonx/Prompt Evaluation with watsonxgov.ipynb [hunk 1]
```diff
@@ -157,7 +182,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In order to set up a development type subscription, the PTA must be within the project. Please supply the project ID where the PTA needs to be created."
+    "In order to set up a development type subscription, the PTA must be within the project. Please supply the project ID where the PTA needs to be created. Your OpenScale instance must be associated with this project"
```
Comment (ravichamarthy) [inline IBM Cloud/WML/notebooks/watsonx/Prompt Evaluation with watsonxgov.ipynb:185]: `Your OpenScale instance must be associated with this project` is untrue. We associate WML instant and not OpenScale instance

**Review by ravichamarthy — APPROVED** [review]
Looks good to me, with minor comment. modify it and merge. Thanks

**Comment by kishore-patel** [comment]
@Hari-Krishnan3 - We need to do the instance mapping when user is using the CP4D env ? Look at my comment https://github.ibm.com/aiopenscale/tracker/issues/33955#issuecomment-72018883

cc: @ravichamarthy

**Comment by Hari-Krishnan3** [comment]
> @Hari-Krishnan3 - We need to do the instance mapping when user is using the CP4D env ? Look at my comment https://github.ibm.com/aiopenscale/tracker/issues/33955#issuecomment-72018883
> 
> cc: @ravichamarthy

Ok Kishore..and from which version of SDK this changes are available ?

**Comment by kishore-patel** [comment]
Currently changes are on test pypi. You can install it locally from SDK repo and test it out (test pypi doesn't have mac version). Once the SDK is released, we can release the notebook too. We plan to start releasing SDK on Monday.

**Comment by Hari-Krishnan3** [comment]
> Currently changes are on test pypi. You can install it locally from SDK repo and test it out (test pypi doesn't have mac version). Once the SDK is released, we can release the notebook too. We plan to start releasing SDK on Monday.

Ok. Thank you

**Comment by Hari-Krishnan3** [comment]
> Currently changes are on test pypi. You can install it locally from SDK repo and test it out (test pypi doesn't have mac version). Once the SDK is released, we can release the notebook too. We plan to start releasing SDK on Monday.

Hi Kishore, Do we have this changes in SDK now ?

**Comment by kishore-patel** [comment]
@Hari-Krishnan3 - SDK changes will be released to public pypi today to latest by tomorrow

**Comment by Hari-Krishnan3** [comment]
> @Hari-Krishnan3 - SDK changes will be released to public pypi today to latest by tomorrow

Ok. Thank you

**Pull request merged**